src/components/questions/ImageUpload.tsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import {
+  StyleSheet,
+  Text,
+  View,
+  TouchableOpacity,
+  Image,
+  ToastAndroid,
+  Alert,
+  PermissionsAndroid,
+} from 'react-native';
+import { launchCamera, launchImageLibrary } from 'react-native-image-picker';
+import Icon from 'react-native-vector-icons/FontAwesome';
+import COLORS from '../../assets/colors';
+
+const ImageUpload = () => {
+  const [image, SetImage] = React.useState('');
+
+  // Show toast alert messages
+  const setToastMsg = (msg: any) => {
+    ToastAndroid.showWithGravity(msg, ToastAndroid.SHORT, ToastAndroid.CENTER);
+  };
+
+  const uploadImage = () => {
```

Comment:
Zou dit niet uploadImage noemen, er wordt helemaal geen image geupload. Je kiest hier een image uit je galerij. PickImageFromGallery of iets in die geest